docs/docs/reference/crd-reference/config.md

```diff
@@ -39,17 +39,26 @@ spec:
       to put into the same auto-generated [KeptnApp](app.md).
       The default value is 30 (seconds).
     * **cloudEventsEndpoint** -- Endpoint where the lifecycle operator posts Cloud Events.
+    * **blockDeployment** -- An option used to block the deployment of the application until the
```

Comment:
This field also needs to be added to the Synopsis section at the top of the page, and the description should link to the page  you wrote in the "Components" section.  A link to that page should also be added to the "See also" list at the bottom of te page